app/views/users/confirmations/new.html.erb

```diff
@@ -5,7 +5,7 @@
 
   <div class="field">
     <%= f.label :email %><br />
```

Comment:
@lilshim I think this was a result of fixing a merge conflict; I replaced the HEAD>>>> stuff with the MASTER>>>> stuff because I thought that MASTER>>>> stuff was the most up-to-date. I never actually meant to edit this file.